src/kernels/conv1x1u_bias_activ.s

```diff
@@ -361,6 +361,9 @@ bias_buffer_size = (output_channels + elements_in_dword - 1) / elements_in_dword
     .VGPR_ALLOC tid
     .VGPR_ALLOC voffset_in
     .VGPR_ALLOC voffset_out
+    .if (.amdgcn.gfx_generation_number == 9 && .amdgcn.gfx_generation_stepping == 10)
+    .VGPR_ALLOC valign_unused, (.VGPR_NEXT_FREE % 2) // inputA/B must be 64 bit aligned
```

Comment:
[Recommendation] The code itself says that that `inputA/B` are 64-bit aligned for gfx90a, no comment is needed to explain that. The comment should explain _why_ this alignment is required. The previous version of comment was better in this regard.